internal/service/eks/addon_test.go

```diff
@@ -912,3 +944,15 @@ resource "aws_eks_addon" "test" {
 }
 `, rName, addonName, tagKey1, tagValue1, tagKey2, tagValue2))
 }
+
+func testAccAddonConfig_configurationValues(rName, addonName, addonVersion, configurationValues string) string {
+	return acctest.ConfigCompose(testAccAddonBaseConfig(rName), fmt.Sprintf(`
+resource "aws_eks_addon" "test" {
+  cluster_name 			= aws_eks_cluster.test.name
```

Comment:
The inline HCL formatting appears off here. We use spaces, not tabs, in the inline HCL.